tests/WE2E/test_configs/grids_extrn_mdls_suites_community/config.grid_RRFS_CONUScompact_25km_ics_RRFS_lbcs_RRFS_suite_RRFS_v1beta.yaml

```diff
@@ -0,0 +1,34 @@
+metadata:
+  description: |-
+    This test is to ensure that the workflow running in community mode
+    completes successfully on the RRFS_CONUScompact_25km grid using the RRFS_v1beta
+    physics suite. It uses RRFS forecasts mapped onto 3-km regular grid (rrfs*.conus.grib2) for
+    ICs and LBCs. This test uses old v1 sfc_data, not the v2 fractional grid sfc_data.
+user:
+  RUN_ENVIR: community
+platform:
+  EXTRN_MDL_DATA_STORES: aws
+workflow:
+  CCPP_PHYS_SUITE: FV3_RRFS_v1beta
+  PREDEF_GRID_NAME: RRFS_CONUScompact_25km
+  DATE_FIRST_CYCL: '2024060517'
+  DATE_LAST_CYCL: '2024060517'
+  FCST_LEN_HRS: 3
+  PREEXISTING_DIR_METHOD: rename
+task_get_extrn_ics:
+  EXTRN_MDL_NAME_ICS: RRFS
+  FV3GFS_FILE_FMT_ICS: grib2
+  USE_USER_STAGED_EXTRN_FILES: false
+task_get_extrn_lbcs:
+  EXTRN_MDL_NAME_LBCS: RRFS
+  LBC_SPEC_INTVL_HRS: 1
+  FV3GFS_FILE_FMT_LBCS: grib2
+  USE_USER_STAGED_EXTRN_FILES: false
+task_plot_allvars:
+  COMOUT_REF: ""
+global:
+  DO_ENSEMBLE: true
+  NUM_ENS_MEMBERS: 2
```

Comment:
Does this still need to be an ensemble after all the other parts were stripped from the PR?

Reply:
An ensemble may not be necessary, after the plot.yaml file was updated through a different PR. It could be changed to DO_ENSEMBLE: false

Reply:
@MichaelLueken - would it be OK to remove the three lines from the new test configuration, as Christina requested above, without affecting the testing procedure?